### DISCUSSION TITLE
Welcome to DUCKSPLOIT Discussions!

### DISCUSSION THREAD

**Discussion by canarddu38** [canarddu38/DUCKSPLOIT#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by canarddu38** [comment]
### DUCKSPLOIT V1.0.8 is coming soon 🥳 🎉

**Comment by Mrpolack** [comment]
Hi, i just need clarification on the following steps:

"Move CircuitPython.uf2 file in the E:\ or F:\ <-- usb port (port of your pico) now your raspberry pico gonna leave usb port and reconnec with CIRCUITPY name"

The "CircuitPython.uf2" is 1,44 mb, and the pico memory is 1,00 MB, so how am I supposed to move that?

And in the next step "copy all the extracted repo folder into him" is that referred only to the "client" folder of the repo?

**Reply by canarddu38** [reply]
### Hi, 
**@Mrpolack DUCKSPLOIT is under developement ,** 
so it's not avaliabe totay because we're working on the **V1.0.8**
But you can fix that by deleting images, idea, docs and all big folders  **BUT NOT CLIENT & HOST folders**
Sorry for the bug 😄
